src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.utils.ts

```diff
@@ -57,6 +57,7 @@ const shiftAnswerValues = (answers: Answer[]) =>
     ...item,
     answer: {
       ...item.answer,
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
no need eslint exceptions here, please enhance a bit the type assertion within isValueDefined:
```
export const isValueDefined = <T = string | number | (string | number)[] | null>(value?: T): value is NonNullable<T>  =>
  value !== null && value !== undefined;
```